app/javascript/orangelight/orangelight_ui_loader.es6

```diff
@@ -24,6 +26,20 @@ export default class OrangelightUiLoader {
     au2.scsb_search_availability();
   }
 
+  setup_linked_records() {
+    const buttons = document.querySelectorAll('.show-more-linked-records');
+    if (buttons.length > 0) {
+      for (let i in buttons) {
```

Comment:
@sandbergja can you change this block https://github.com/pulibrary/orangelight/pull/3101/files#diff-b5edfa9e9fcc753de1d2d0bd284c0952bafc00589ab342edd2162748fc8fc4d0R32-R36 to:
```
for (let button of buttons) {
        const fetchData = RelatedRecordsDisplayer.fetchData(
          button.getAttribute('data-linked-records-field'),
          button.getAttribute('data-record-id'));
        button.addEventListener('click', (event) => {
          fetchData.then(displayer => displayer.toggle(event))
        });
      }  
``` as it is it throws an error : `Uncaught TypeError: buttons[i].getAttribute is not a function
    at OrangelightUiLoader.setup_linked_records (orangelight_ui_loader.es6:35:1)
    at OrangelightUiLoader.run`